### ISSUE TITLE
Map tiles are incorrectly importing to Blender

### ISSUE THREAD

**Issue by nxrighthere** [Kruithne/wow.export#131]
It seems that map tiles are screwed with all WMOs and Doodads in Blender using the plugin.

**Comment by Kruithne** [comment]
We're going to need a far better explanation than that before we can look into this issue. Please provide more details, examples, screenshots or logs.

**Comment by nxrighthere** [comment]
So I just export 4 selected tiles in _Maps_ section with all options checked on the right bar:
![x](https://user-images.githubusercontent.com/8279330/80822365-03304e80-8bf4-11ea-9fdd-c960af44504c.PNG)
Then import one of the tiles to Blender using the plugin, and focus on:
![z](https://user-images.githubusercontent.com/8279330/80822826-f3653a00-8bf4-11ea-97e4-5eda70d25617.PNG)
As you can see, WMOs and Doodads are imported, but not located correctly.

**Comment by Kruithne** [comment]
Can you please confirm you are using the latest version of the Blender plug-in that we provide with wow.export itself? You can click the Blender icon in the top-right to install the latest version.

If you have multiple installations of Blender, it is possible it did not install to the correct one, in which case you may need to manually do so. Please confirm the current version of the add-on you have enabled in Blender for me.

**Comment by nxrighthere** [comment]
Yes, it was the first thing that I've checked, the current installed version 0.3.4.

I have only one Blender installation (2.82 version), tired to install the plugin manually with clearing the cache and still the same thing, unfortunately.

**Comment by Kruithne** [comment]
I'm having trouble re-producing this. @Marlamin any input?

**Comment by Marlamin** [comment]
No idea, really. Some shots in the dark:

What game version are you exporting this out of? 
Are you using local/online mode?

**Comment by nxrighthere** [comment]
I'm using local installation, 8.3.0.34220 version. I've tried to re-download toolkit from the website and update it, still the same.

**Comment by nxrighthere** [comment]
Tried fresh Blender installation wiping out all options, and restoring the client data - doesn't help.

**Comment by Marlamin** [comment]
Yeah, I really have no idea what this could be. :(

**Comment by nxrighthere** [comment]
M2 models are exporting/importing correctly. 🤔

**Comment by Kruithne** [comment]
Can you wipe your entire export folder, export a single tile, and then ZIP and upload your export folder somewhere so I can take a look at the output?

**Comment by nxrighthere** [comment]
Sure, [here's](https://drive.google.com/file/d/1dv_0KoMJ2ue4OQXpKOwqVRKm4nQNDYnX/view?usp=sharing) Eastern Kingdoms (31 49).

**Comment by Kruithne** [comment]
![Blender_2020-05-02_11-11-27](https://user-images.githubusercontent.com/682348/80861303-af5e4d80-8c65-11ea-904b-de8ee05096f8.png)

This imports absolutely fine for me. The paths are relative, so it should work regardless, but can you just confirm for me the full path of your export directory?

**Comment by nxrighthere** [comment]
I'm importing from `E:\wowexport\` [here's](https://drive.google.com/file/d/11gIZMeLhxr6W9-YMjlZbH2of6nEQ1T7P/view?usp=sharing) the video of the process.

**Comment by RomanZhu** [comment]
^ I have the same problem on the blender 2.82 with the latest plugin

**Comment by nxrighthere** [comment]
So far what I've tried, and it doesn't work:
- Downgrading to Blender 2.80
- Changing unit scale of the scene before the import
- Wiping out all settings of Blender and reinstalling it

**Comment by nxrighthere** [comment]
It's just something wrong with positions. Individual objects imported correctly, but for some reason when the plugin is trying to combine them with a tile, they scattered all over the scene.

**Comment by Marlamin** [comment]
This is very confusing, installed a fresh Blender/wow.export/plugin and unable to reproduce this issue. Can you save a .blend (with everything packed) and upload it?

![blender_J9Gl1jPqxv](https://user-images.githubusercontent.com/50212/81105249-eeb1c600-8f13-11ea-84f3-904fba81df90.png)

I'd like to take a look at how it imported it for you.

**Comment by nxrighthere** [comment]
Sure, [here's](https://drive.google.com/open?id=1zvY7dUnMRG7Y4t2zAcMQy-8lx0xOqhJ_) the packed scene.

**Comment by Marlamin** [comment]
That definitely looks imported incorrectly, but not sure in what way exactly.

Funnily enough, importing something into that .blend is just fine, so it's not some kind of setting or something. I'm really drawing blanks here. Maybe something environment/locale based? What regions are both of you from (and as such, what Windows locale settings are you on)?

![image](https://user-images.githubusercontent.com/50212/81114338-26c00580-8f22-11ea-93b7-8587925dad4b.png)

**Comment by nxrighthere** [comment]
So my OS locale is English (United States) and the game client is English (EU).

**Comment by nxrighthere** [comment]
I just recently changed it from Russian, no effect.

**Comment by nxrighthere** [comment]
Might be something related to encoding UTF-8/Unicode?

**Comment by RomanZhu** [comment]
OS locale is English (United States)  (but I have the Russian language installed too)
game client is English (EU)

**Comment by Deams51** [comment]
Getting the same issue here.
WoW: EU EN
OS: US
Blender: 2.91
WoW.Export + plugin: Current release (tried master as well)
